### DISCUSSION TITLE
Positive RSSI values

### DISCUSSION THREAD

**Discussion by kmfnjtze8888** [openthread/openthread#7705] [Q&A] [answered]
Hello,

How are RSSI values calculated for OpenThread? When I use the `neighbor table` command, the **Last RSSI** values for some of the neighbors are positive, and some are negative. I thought RSSI values here are always negative?

**Comment by jwhui** [comment, marked as answer]
RSSI values are provided by the radio driver - see [nRF52840 example](https://github.com/openthread/ot-nrf528xx/blob/da1425c26dd9e5968195ef9b01deede30b22a115/src/src/radio.c#L1011). If you have questions about the RSSI values, I suggest reaching out to the IEEE 802.15.4 silicon vendor.
